generators/client/templates/react/src/test/javascript/spec/app/modules/account/password/password.reducer.spec.ts.ejs

```diff
@@ -60,4 +65,35 @@ describe('Password reducer tests', () => {
       });
     });
   });
+
+  describe('Actions', () => {
+    let store;
+
+    const resolvedObject = { value: 'whatever' };
+    beforeEach(() => {
+      const mockStore = configureStore([thunk, promiseMiddleware()]);
+      store = mockStore({});
+      axios.post = sinon.stub().returns(Promise.resolve(resolvedObject));
+    });
+
+    it('dispatches UPDATE_PASSWORD_PENDING and UPDATE_PASSWORD_FULFILLED actions', async () => {
+      const meta = {
+        errorMessage: '<strong>An error has occurred!</strong> The password could not be changed.',
+        successMessage: '<strong>Password changed!</strong>'
+      };
+
+      const expectedActions = [
+        {
+          type: REQUEST(ACTION_TYPES.UPDATE_PASSWORD),
+          meta
+        },
+        {
+          type: SUCCESS(ACTION_TYPES.UPDATE_PASSWORD),
+          payload: resolvedObject,
+          meta
+        }
+      ];
+      await store.dispatch(savePassword('', '')).then(() => expect(store.getActions()).to.eql(expectedActions));
```

Comment:
@deepu105 There's something weird, when this test is run with i18n enabled, it fails because of translation error but they're just fine when it's run out of the test. Any idea why ?
```
FAILED TESTS:
  Password reducer tests
    Actions
      ✖ dispatches UPDATE_PASSWORD_PENDING and UPDATE_PASSWORD_FULFILLED actions
        HeadlessChrome 67.0.3372 (Linux 0.0.0)
      AssertionError: expected [ Array(2) ] to deeply equal [ Array(2) ]
      + expected - actual
       [
         {
           "meta": {
      -      "errorMessage": "translation-not-found[password.messages.error]"
      -      "successMessage": "translation-not-found[password.messages.success]"
      +      "errorMessage": "<strong>An error has occurred!</strong> The password could not be changed."
      +      "successMessage": "<strong>Password changed!</strong>"
           }
           "type": "account/UPDATE_PASSWORD_PENDING"
         }
         {
           "meta": {
      -      "errorMessage": "translation-not-found[password.messages.error]"
      -      "successMessage": "translation-not-found[password.messages.success]"
      +      "errorMessage": "<strong>An error has occurred!</strong> The password could not be changed."
      +      "successMessage": "<strong>Password changed!</strong>"
           }
           "payload": {
             "value": "whatever"
           }
      
    at eval (webpack:///./src/test/javascript/spec/app/modules/account/password/password.reducer.spec.ts?:79:165)
```

Reply:
Thats coz i18n files are not loaded in tests I guess, may be we could just check for the no translation message as it has the key